src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -319,11 +333,16 @@ def _configure_db_create_params(
     arg_ctx.argument('min_capacity',
                      arg_type=min_capacity_param_type)
 
+    arg_ctx.argument('read_scale',
```

Comment:
I thought we decided that these should go into `sql db` context so that they apply to both `sql db create` and `sql db update` ?

Reply:
Isn't that where they are?  I removed them from lower down sql db update.  They are working for both create and update.  Maybe the name _configure_db_create_params is misleading?  It seems to be intended and working for both create and update: "Configures params for db/dw create/update commands."